### ISSUE TITLE
Questions about inpaint training code

### ISSUE THREAD

**Issue by universewill** [lucidrains/DALLE2-pytorch#249]
Hi, i don't quite understand the inpainting training framework.

1.  Can i use a pretrained clip model?
2.  Is only the decoder need to be trained?
3.  What does the "mock_image_embed" mean below? Is this embed need to be generated from clip and diffusion prior model?

'''
inpainted_images = decoder.sample(
    image_embed = mock_image_embed,
    inpaint_image = inpaint_image,    # just pass in the inpaint image
    inpaint_mask = inpaint_mask       # and the mask
)
'''

**Comment by zhangchen234** [comment]
I have the same question.
